tyk-control-plane/README.md

```diff
@@ -0,0 +1,793 @@
+
+> To explore the list of supported backends for Tyk Pump, please visit https://tyk.io/docs/tyk-stack/tyk-pump/other-data-stores/.
+
+#### Prometheus Pump
+Add `prometheus` to `tyk-pump.pump.backend`, and add connection details for Prometheus under `tyk-pump.pump.prometheusPump`.
+
+We also support monitoring using Prometheus Operator. All you have to do is set `tyk-pump.pump.prometheusPump.prometheusOperator.enabled` to true.
+
+This will create a _PodMonitor_ resource for your Pump instance.
+
+```yaml
+    # prometheusPump configures Tyk Pump to expose Prometheus metrics.
+    # Please add "prometheus" to .Values.pump.backend in order to enable Prometheus Pump.
+    prometheusPump:
+      # host represents the host without port, where Tyk Pump serve the metrics for Prometheus.
+      host: ""
+      # port represents the port where Tyk Pump serve the metrics for Prometheus.
+      port: 9090
+      # path represents the path to the Prometheus collection. For example /metrics.
+      path: /metrics
+      # customMetrics allows defining custom Prometheus metrics for Tyk Pump.
+      # It accepts a string that represents a JSON object. For instance,
+      #
+      # customMetrics: '[{"name":"tyk_http_requests_total","description":"Total of API requests","metric_type":"counter","labels":["response_code","api_name","method","api_key","alias","path"]},          {              "name":"tyk_http_latency",              "description":"Latency of API requests",              "metric_type":"histogram",              "labels":["type","response_code","api_name","method","api_key","alias","path"]          }]'
+      customMetrics: ""
+      # If you are using prometheus Operator, set the fields in the section below.
+      prometheusOperator:
+        # enabled determines whether the Prometheus Operator is in use or not. By default,
+        # it is disabled.
+        # Tyk Pump can be monitored with PodMonitor Custom Resource of Prometheus Operator.
+        # If enabled, PodMonitor resource is created based on .Values.pump.prometheusPump.prometheusOperator.podMonitorSelector
+        # for Tyk Pump.
+        enabled: false
+        # podMonitorSelector represents a podMonitorSelector of your Prometheus resource. So that
+        # your Prometheus resource can select PodMonitor objects based on selector defined here.
+        # Please set this field to the podMonitorSelector field of your monitoring.coreos.com/v1
+        # Prometheus resource's spec.
+        #
+        # You can check the podMonitorSelector via:
+        #   kubectl describe prometheuses.monitoring.coreos.com <PROMETHEUS_POD>
+        podMonitorSelector:
+          release: prometheus-stack
+```
+
+#### Mongo pump
+If you are using the MongoDB pumps in the tyk-oss installation you will require MongoDB installed for that as well.
+
+To install MongoDB you can use these rather excellent charts provided by Bitnami:
```

Comment:
We already have documented installation steps for mongo and postgres at the beginning. Can we simply point to it instead of defining it here again?